ts/features/itwallet/machine/credential/machine.ts

```diff
@@ -58,7 +59,9 @@ export const itwCredentialIssuanceMachine = setup({
     isSessionExpired: notImplemented,
     isDeferredIssuance: notImplemented,
     hasValidWalletInstanceAttestation: notImplemented,
-    isStatusError: notImplemented
+    isStatusError: notImplemented,
+    isWalletValid: notImplemented,
```

Comment:
Does this guard also check for the eID jwt expiration? Because if I mock the eID status to `jwtExpired` this guard returns `true`.
In fact, it calls `itwLifecycleIsValidSelector`, which does not take in consideration the eID expiration and, in my opinion, it is not the correct behavior.